api-client/src/main/java/com/takipi/api/core/url/UrlClient.java

```diff
@@ -51,7 +109,13 @@ public String getHostname() {
 			connection.setReadTimeout(readTimeout);
 			connection.setRequestMethod("GET");
 
-			return getResponse(targetUrl, connection);
+			long t1 = System.currentTimeMillis();
```

Comment:
Why not do this entire time wrapping inside `getResponse` in a `finally` block?
Reduces unnecessary code duplication

Note that `targetUrl` is different then `url.toString()`, but given that `getResponse` only uses the given parameter for logging, I'm ok with it getting `url.toString` instead of `targetUrl`